CHANGELOG.md

```diff
@@ -94,7 +100,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
-- Operator-rs: `0.42.2` -> `0.44.0` ([#452]).
+- Operator-rs: `0.42.2` -> `0.44.0` ([#434], [#452]).
```

Comment:
nit(non-blocker): I think we should get rid of the "from" version. Next time, eh?

Reply:
Agreed, I just added the link back in, because `markdownlink` was complaining with:

```
CHANGELOG.md:114:1 MD053/link-image-reference-definitions Link and image reference definitions should be needed [Unused link or image reference definition: "#434"] [Context: "[#434]: https://github.com/sta..."]
```